src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.html

```diff
@@ -62,55 +77,377 @@ <h3 class="u-text-body-bold">Badge Basics</h3>
+					<div
+						style="
+							border: 2px solid;
+							border-radius: 25px;
+							border-color: var(--color-blue);
+							padding: 2em;
+							margin-top: 2em;
+						"
+					>
+						<div style="display: flex; justify-content: space-around">
+							<bg-formfield-text
+								[control]="competency.rawControlMap.name"
+								label="Kompetenz-Titel"
+								[fieldType]="'text'"
+								[errorMessage]="{
+									required: 'Bitte gib den Titel der Kompetenz ein.'
+								}"
+							></bg-formfield-text>
+							<bg-formfield-text
+								[control]="competency.rawControlMap.studyLoad"
+								label="Dauer Kompetenz- Erwerb (in Minuten)"
+								[fieldType]="'number'"
+								[errorMessage]="{
+									required: 'Bitte gib die Dauer für den Erwerb der Kompetenz ein.'
+								}"
+							></bg-formfield-text>
+							<bg-formfield-select
+								ariaLabel="Wähle eine Kompetenz-Kategorie"
+								[control]="competency.rawControlMap.category"
+								[label]="'Kategorie'"
+								[placeholder]="'Kategorie wählen'"
+								[optionMap]="competencyCategoryOptions"
+								[errorMessage]="{
+									required: 'Bitte wähle die Kategorie der Kompetenz aus.'
+								}"
+							></bg-formfield-select>
+						</div>
+						<bg-formfield-text
+							style="margin-left: 2em; margin-top: 2em; margin-right: 2em"
+							[control]="competency.rawControlMap.description"
+							label="Kompetenz Beschreibung"
+							id="competency_description_input"
+							placeholder="Kurze Beschreibung der Kompetenz"
+							[errorMessage]="{ required: 'Bitte gib eine Beschreibung ein' }"
+							[multiline]="true"
+						></bg-formfield-text>
+						<bg-formfield-text
+							style="margin-left: 2em; margin-top: 2em; margin-right: 2em"
+							[control]="competency.rawControlMap.escoID"
+							label="Esco Identifier"
+							id="competency_description_input"
+							placeholder="z.B. esco/skill/00090cc1-1f27-439e-a4e0-19a87a501bfc"
+						></bg-formfield-text>
+						<div class="l-flex l-flex-1x l-flex-justifybetween u-margin-bottom2x" style="margin-top: 2em">
+							<button
+								class="button"
+								type="button"
+								style="background-color: var(--color-grün); color: var(--color-blue)"
+							>
+								Hinzufügen
+							</button>
+							<button
+								*ngIf="badgeClassForm.controls.competencies.controls.length > 0"
+								type="button"
+								(click)="removeCompetency(competency)"
+								class="u-text-link"
+							>
+								Remove
+							</button>
+						</div>
+						<!-- <button
+							class="button"
+							style="background-color: var(--color-grün); color: var(--color-blue); margin-top: 2em"
+						>
+							Hinzufügen
+						</button> -->
+					</div>
+				</ng-container>
+				<!-- </ng-template> -->
+				<div class="l-stack l-stack-right u-margin-top2x">
+					<button (click)="addCompetency()" type="button" class="button button-secondary">
+						Kompetenz hinzufügen
```

Comment:
There seem to be two "Hinzufügen" buttons

Reply:
I commented one out for now, will clarify which one we should use